src/server/api/medias/musicres.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+const router = express.Router();
+
+const MusicresServices = require('../../services/MusicresServices');
+
+router.get('/', (req, res) => {
+  const roles = res.locals.token.roles.map(role =>
```

Comment:
@maxpluez This is violating the DRY principle: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

All this repeated code means there is need for a function. See @kilometerskang made a file to contain this logic: https://github.com/ucla/lti-library-resources/blob/70c423ef086fd6cf0ac96c9f5af288f1b6c79b75/src/client/components/App/userRoles.js#L3